test/src/comps/test_orth.cc

```diff
@@ -1,17 +1,14 @@
-/*
-TODO #1: Switch tuples to vectors.
-*/
+#include "RandBLAS.hh"
+#include "RandLAPACK.hh"
```

Comment:
I think RandBLAS should be included with brackets.

Reply:
done